lib/Client.js

```diff
@@ -601,8 +602,13 @@ class Client extends EventEmitter {
                 };
             } else if(content.content !== undefined && typeof content.content !== "string") {
                 content.content = "" + content.content;
-            } else if(content.content === undefined && !content.embed && !file) {
-                return Promise.reject(new Error("No content, file, or embed"));
+            } else if(content.content === undefined && !content.embed && !content.embeds && !file) {
+                return Promise.reject(new Error("No content, file, or embeds"));
+            } else if(content.embed && content.embeds) {
+                content.embeds.push(content.embed);
+            } else if(content.embed) {
```

Comment:
No, just check if embed exists and if embeds doesn't exist, and if both those conditions are true, then set content.embeds to [content.embed]
I'm really not sure why you're complicating things by pushing (which will break anyways if someone puts the wrong thing in embeds)
Also deprecation will not get captured in this case, hence my request